src/workerd/server/tests/python/sdk/mock.py

```diff
@@ -0,0 +1,17 @@
+from cloudflare.workers import Response
+
+
+async def on_fetch(request):
+    if request.url.endswith("/sub"):
+        return Response("Hi there!")
+    elif request.url.endswith("/sub_headers"):
+        assert request.headers.get("Custom-Req-Header") == "123"
+        return Response(
+            "Hi there!", headers={"Custom-Header-That-Should-Passthrough": True}
+        )
+    else:
+        raise AssertionError("Unexpected path " + request.url)
```

Comment:
It's a bit odd to raise an `AssertionError` on something that came across the application boundary like this. A `ValueError` might be most appropriate here.

Reply:
fwiw this was a `assert False` but Ruff complained and suggested using `raise AssertionError` so I followed its advice. I don't think the exact error matters much here since this is a test, but I'll change it.